featuremanagement/_featuremanager.py

```diff
@@ -87,13 +92,22 @@ def get_variant(self, feature_flag_id, *args, **kwargs):
         targeting_context = self._build_targeting_context(args)
 
         result = self._check_feature(feature_flag_id, targeting_context, **kwargs)
-        if self._on_feature_evaluated and result.feature.telemetry.enabled:
+        if (
+            self._on_feature_evaluated
+            and result.feature
+            and result.feature.telemetry.enabled
+            and callable(self._on_feature_evaluated)
+        ):
             result.user = targeting_context.user_id
             self._on_feature_evaluated(result)
         return result.variant
 
-    def _check_feature_filters(self, evaluation_event, targeting_context, **kwargs):
+    def _check_feature_filters(
+        self, evaluation_event: EvaluationEvent, targeting_context: TargetingContext, **kwargs: Dict
+    ) -> None:
```

Comment:
Is this new line intended?

Reply:
Yes, the line with the type hints is now over 120 characters, so our formatter requires it to be on a new line.